app/views/unattended/provisioning_templates/PXEGrub2/preseed_default_pxegrub2.erb

```diff
@@ -15,22 +15,12 @@ description: |
 #
 # This file was deployed via '<%= template_name %>' template
 
-<%
-  os_major = @host.operatingsystem.major.to_i
-  os_name  = @host.operatingsystem.name
-
-  if (os_name == 'Ubuntu' && os_major > 12) || (os_name == 'Debian' && os_major > 8)
-    efi_suffix = 'efi'
-  else
-    efi_suffix = ''
-  end
--%>
 set default=0
 set timeout=<%= host_param('loader_timeout') || 10 %>
 
 menuentry '<%= template_name %>' {
-  linux<%= efi_suffix %>  <%= @kernel %> interface=auto url=<%= foreman_url('provision')%> ramdisk_size=10800 root=/dev/rd/0 rw auto BOOTIF=01-$net_default_mac <%= snippet("preseed_kernel_options").strip %>
-  initrd<%= efi_suffix %> <%= @initrd %>
+  linux <%= @kernel %> interface=auto url=<%= foreman_url('provision')%> ramdisk_size=10800 root=/dev/rd/0 rw auto BOOTIF=01-$net_default_mac <%= snippet("preseed_kernel_options").strip %>
```

Comment:
Why is the `efi` suffix not needed anymore?
From the removed code above I can see that ubuntu >12  and debian >8 had to have `efi` suffix. What changed?